### ISSUE TITLE
custom mailer question

### ISSUE THREAD

**Issue by royzinn** [plataformatec/devise#1982]
Hi,
I read you HOW TO regarding using custom mailer and I have a question.
I want to use Mandrill for sending the emails and I need to add some headers to the confirmation/reset email.
should I override the functions confirmation_instructions(record) & reset_password_instructions(record)? 
Can i define them on my mailer, call super and add my stuff?
Do I need to have the views templates under my mailer views or can I still use Devise's views?

Basically, all I want to do is add headers['X-MC-Track'] = "opens, clicks_htmlonly" for example so that I can trace Devise emails...

Thanks,
Roy
